engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java

```diff
@@ -934,4 +934,8 @@
 
 	@Message(id = 305, value = "Analyzer reference with name '%2$s' not initialized for field '%1$s' ")
 	SearchException analyzerReferenceNotInitialized(String fieldName, String name);
+
+	@Message(id = 306, value = "IndexManager type is required to build a FieldBridge for %2$s in %1$s")
```

Comment:
Given it's meant to be read by users, I'd avoid the camel-casing: "Index manager", "field bridge".